kotishell/include/minilibft.h

```diff
@@ -55,4 +55,9 @@ char		*mini_concat(const char *s1, const char *s2);
 */
 void		*mini_free(void *p);
 
+/**
+ * @brief joins 2 strings and returns the resulted string
```

Comment:
Maybe add more useful info? That the strings must be null-terminated, not-null, and that the new string is allocated on heap and the old ones aren't freed and on malloc error a NULL is returned?

Reply:
updated